spec/support/verifiable.rb

```diff
@@ -179,4 +179,85 @@
     end
   end
 
+  describe "methods modified by Setting user.skip_verification" do
+
+    let(:user) {create(:user)}
+
+    before do
+      Setting["feature.user.skip_verification"] = 'true'
+    end
+
+    after do
+      Setting["feature.user.skip_verification"] = nil
+    end
+
+    describe "#residence_verified?" do
+      it "is true if skipped" do
+        expect(user.residence_verified?).to eq(true)
+      end
+    end
+
+    describe "#sms_verified?" do
+      it "is true if skipped" do
+        expect(user.sms_verified?).to eq(true)
+      end
+    end
+
+    describe "#level_two_verified?" do
+      it "is true if skipped" do
+        expect(user.level_two_verified?).to eq(true)
+
+        user.update(residence_verified_at: Time.current)
+        expect(user.level_two_verified?).to eq(true)
+
+        user.update(confirmed_phone: "123456789", residence_verified_at: false)
+        expect(user.level_two_verified?).to eq(true)
+      end
+    end
+
+    describe "#level_three_verified?" do
+      it "is true if skipped" do
+        expect(user.level_three_verified?).to eq(true)
+      end
+    end
+
+    describe "#unverified?" do
```

Comment:
Please take your time to read my comments again, `unverified?` has nothing to do neither its affected with the verification skip feature. Why are we testing that by default it returns false here?

Reply:
I'll remove that test, but that function it's affected by the skip setting.
Its defined 
```
def unverified?
  !level_two_or_three_verified?
end
```
and level_two_or_three_verified? like 
```
def level_two_or_three_verified?
  level_two_verified? || level_three_verified?
end
```
also affected by the skip setting.

I thought that I should test any function affected by my changes